### DISCUSSION TITLE
Permission denied to laravel.log in Lightsail server

### DISCUSSION THREAD

**Discussion by alexugthub** [deployphp/deployer#2288] [Help needed]
I am trying to use Deployer along with the [Laravel](https://github.com/deployphp/deployer/blob/master/recipe/laravel.php) recipe.

Everything works except I get this reported back from the pipeline:

```
➤ Executing task artisan:optimize
[mydomain.com] > /opt/bitnami/php/bin/php ~/htdocs/develop/releases/1/artisan optimize
[mydomain.com] < Configuration cache cleared!
[mydomain.com] < In StreamHandler.php line 111:
[mydomain.com] <   The stream or file "/opt/bitnami/apache/htdocs/develop/releases/1/storage/l
[mydomain.com] <   ogs/laravel.log" could not be opened in append mode: failed to open stream:
[mydomain.com] <    Permission denied
```

I have two users: _bitnami_ (SSH) and _daemon_ (Apache). I need user daemon to have permission for the _storage_ folder and I did this inside _deployer.php_

```
set('writable_use_sudo', true);
set('clear_use_sudo', true);
set('cleanup_use_sudo', true);
set('writable_mode', 'chown');
set('http_user', 'daemon');
```

What I see is that folder _releases/1/storage/logs_ has user _daemon_ assigned to group _bitnami_. I think it should belong to group _daemon_ and I don't know how to make Deployer change the group for the _daemon_ user

**Comment by antonmedv** [comment]
This is in writable recipe. I think this due to selected mode. One solution extend writable recipe to support this, or write you own recipe.

**Comment by alexugthub** [comment]
Looks like changing from `chown` to `chgrp` did the trick. Still giving it a few tests
